region/maridia/inner-pink.json

```diff
@@ -452,7 +305,18 @@
               "unlock": null
             },
             {
-              "id": 6,
+              "id": 7,
+              "requires": [
+                "canSwim",
+                {"canShineCharge": {
+                  "usedTiles": 31
+                }}
+              ],
+              "unlock": null,
+              "note": "This direct link is just for the shinespark"
+            },
+            {
+              "id": 9,
```

Comment:
I'd like to see the canWalljump and all the other ways to get up here.. Same at node 2 to node 5.

Reply:
My opinion has always been that getting walljumps everywhere should be its own undertaking. Most of the places I've put walljumps have been because they were already in and I was unwilling to take them out. I don't think this needs to be part of the already massive undertaking that junction refactor was.